migration/idempotent/001-base.sql

```diff
@@ -283,9 +282,16 @@ BEGIN
                     NEW.table_schema, NEW.table_name);
    EXECUTE format('GRANT SELECT ON TABLE %I.%I TO prom_reader', NEW.table_schema, NEW.table_name);
    EXECUTE format('GRANT SELECT, INSERT ON TABLE %I.%I TO prom_writer', NEW.table_schema, NEW.table_name);
-   EXECUTE format('GRANT SELECT, INSERT, UPDATE, DELETE, TRIGGER ON TABLE %I.%I TO prom_modifier', NEW.table_schema, NEW.table_name);
+   EXECUTE format('GRANT SELECT, INSERT, UPDATE, DELETE ON TABLE %I.%I TO prom_modifier', NEW.table_schema, NEW.table_name);
    EXECUTE format('CREATE UNIQUE INDEX data_series_id_time_%s ON %I.%I (series_id, time) INCLUDE (value)',
                     NEW.id, NEW.table_schema, NEW.table_name);
+  
+   --dynamically created tables are owned by our highest-privileged role, prom_admin
+   --these cannot be made part of the extension since we cannot make them config
+   --tables outside of extension upgrade/install scripts. They cannot be superuser
+   --owned without being part extension since that would prevent dump/restore from
+   --working on any environment without SU privileges (such as cloud).
+   EXECUTE format('ALTER TABLE %I.%I OWNER TO prom_admin', NEW.table_schema, NEW.table_name);
```

Comment:
From the comment, I understand why to pick `prom_admin` over a superuser. Yet, It's not quite clear to me why the `prom_admin` role works while `prom_modifier` doesn't. From what I remember `ALTER TABLE ... OWNER TO` construct itself was a problem on multi-node.

Reply:
we switched prom_modifier->prom_admin not because of multinode but because prom-admin is our most-privileged user and thus it is the user that should own these things. The multinode fix is moving the ownership change to before hypertable creation